contrib/clojure-package/src/org/apache/clojure_mxnet/ndarray.clj

```diff
@@ -91,6 +91,22 @@
   ([start stop]
    (arange start stop {})))
 
+(defn ->ndarray
+  "Creates a new NDArray based on the given n-dimensional
+   float/double vector.
+    `nd-vec`: n-dimensional vector with floats or doubles.
```

Comment:
Do we want to allow also integers here? or other numerical types that can be understood by MXNet?

Reply:
Maybe the Scala functions will not allow it but we can cast to float/double from clojure?

Reply:
currently, scala doesn't allow numerical types other than float/double.

Reply:
usually i've been using `ndarray/as-type` for type conversion instead of doing conversion within clojure. my hunch is `as-type` is faster but i've not verified this.

Reply:
From a user perspective, I think it would be nice to be able to handle `((ndarray/->ndarray [5 -4 3])` and not throw an exception. What do you think about adding a check in the util function to see if any element is an integer and if so, convert it to double. Ex:

```
(if (some int? s)
      (to-array (mapv double s))
      (to-array s))
```